app/model/Metadata.scala

```diff
@@ -15,6 +16,7 @@ case class UpdatedMetadata (
 
 object UpdatedMetadata {
   implicit val metadataRead: Reads[UpdatedMetadata] = (
+    (__ \ "title").read[String] ~
```

Comment:
The `update-metadata` is used by CDS to [set the plutoId](https://github.com/guardian/content_delivery_system/blob/master/CDS/scripts/js_utils/media-atom/media-atom.js#L126-L129). It sends a payload of `{plutoId: 'KP-xxxxx'}`.

Can this be `readNullable[String]` so that cds doesn't need to send the title in the payload too?

Alternatively, we can get rid of the `update-metadata` endpoint, in place of a `set-pluto-id` one?